docs/development.rst

```diff
@@ -481,6 +481,54 @@ QQ-plots of the statistics in question comparing ``ms`` and ``msprime``.
 There are also several "analytical" tests, which compare the distributions of
 values from ``msprime`` with analytical expectations.
 
+****************
+Containerization
+****************
+
+This repo is integrated with Dockerhub and the Docker image will be automatically
+built upon pushes to master on Dockerhub.
```

Comment:
I don't think we should update the Docker image on updates to master. That would mean that we'd have to be very very careful about what we commit to master, and make the process of releasing a bit redundant. Can we instead update the Docker image when we tag a new release?

Reply:
Yes, we can have it automatically update the Docker image with tagged releases and ignore master.